integration-test/pages/login-in-modal.html

```diff
@@ -37,7 +37,7 @@ <h2>Log in</h2>
     }, {once: true})
     window.addEventListener('pointerdown', (e) => {
       if (!dialogEl.contains(e.target)) {dialogEl.setAttribute('hidden', '')}
-    }, {once: true})
+    })
```

Comment:
This was the problem with the test and why it didn't catch the regression. Since the self-closing listener only fired once, the test was passing because this handler ran as we clicked into the field and not when clicking again in the autofill pulldown UI.